src/cascadia/TerminalApp/Pane.h

```diff
@@ -54,6 +54,7 @@ class Pane : public std::enable_shared_from_this<Pane>
     std::shared_ptr<Pane> GetActivePane();
     winrt::Microsoft::Terminal::Control::TermControl GetTerminalControl();
     winrt::Microsoft::Terminal::Settings::Model::Profile GetFocusedProfile();
+    winrt::Microsoft::Terminal::Settings::Model::Profile GetProfile() const { return _profile; }

```

Comment:
To be clear, `GetProfile()` would only return `nullptr` on non-leaf panes, right? Whereas `GetFocusedProfile()` performs a walk until it finds the focused pane, then gets that profile?

Reply:
correct.